examples/winnip_31604_12061_004_120717_L090_CX_07/winnip_31604_12061_004_120717_L090_CX_07.json

```diff
@@ -4,54 +4,40 @@
   "id": "winnip_31604_12061_004_120717_L090_CX_07",
   "properties": {
     "proj:epsg": 4326,
-    "sat:absolute_orbit": 12061,
-    "datetime": "2012-07-17T14:39:45.732157Z"
+    "datetime": "2012-07-17T14:36:47Z"
   },
   "geometry": {
     "type": "Polygon",
     "coordinates": [
       [
         [
-          -97.683,
```

Comment:
@jjfrench you were commenting about the order. Does it matter, given it's a polygon ? 

The values are slightly different (but we also expect that, we saw that the values were slightly different in the annotated files compared to the HDF5 metadata).

Reply:
It matters for some applications, typically polygons should follow the "right-hand" rule

Reply:
I realize that it's obvious not _any_ order works, sorry 😅.


I am finding contradictory answers as to what the 'right hand rule' means here : clockwise or counter clockwise ? 

The GeoJSON spec seems to be saying counterclockwise for our case, so I switched to that : https://datatracker.ietf.org/doc/html/rfc7946#section-3.1.6.

Reply:
Counterclockwise should be correct, I think that the updated geometry looks good.